e2e/Makefile

```diff
@@ -3,26 +3,31 @@ GOFLAGS=-mod=vendor
 export GOFLAGS
 GINKGO=$(GOPATH)/bin/ginkgo
 KUBECTL=/usr/local/bin/kubectl
-KIND_VERSION=0.5.1
+KIND_VERSION=0.6.1
 KUSTOMIZE_VERSION=3.2.0
 
 # If you want to change the Kubernetes version for e2e test, specify this variable from command line.
 # e.g. $ make TEST_KUBERNETES_VERSION=1.14 setup test
-TEST_KUBERNETES_VERSION=1.15
+TEST_KUBERNETES_VERSION=1.16
 
-ifeq ($(TEST_KUBERNETES_VERSION),1.15)
+ifeq ($(TEST_KUBERNETES_VERSION),1.16)
+KUBERNETES_VERSION=1.16.4
```

Comment:
Only 1.16.3 of `kindest/node` is available:
https://hub.docker.com/r/kindest/node/tags